changelogs/fragments/39-ce_modify_the_parse_mode.yml

```diff
@@ -1,2 +1,2 @@
 bugfixes:
-- "ce - Modify the way of parsing NETCONF XML message in ce.py (https://github.com/ansible-collections/community.network/pull/39)."
+- "ce - Modify the way of parsing NETCONF XML message in ce.py (https://github.com/ansible-collections/community.network/pull/39)."
```

Comment:
Please do not modify an existing changelog fragment if the change is unrelated to this PR.